source/upgrade.txt

```diff
@@ -113,15 +113,15 @@ This library version introduces the following breaking changes:
   methods, you can register a :php:`CommandSubscriber
   <mongodb-driver-monitoring-commandsubscriber>` in your application.
 
-Version 1.20 Breaking Changes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Version 1.x Breaking Changes
```

Comment:
```suggestion
Version 1.x Compatibility Changes
```

We don't consider bumping dependencies such as minimum PHP or server versions as "breaking changes", as that would require us to only do these in a major version. The rationale is that for PHP versions, the compatibility is checked by composer, so users on a version of PHP that is no longer supported won't be upgraded to the newer driver version and thus don't encounter a breaking change. Since the server version drops are so far behind (one year after it's marked EOL by the server team), the number of people affected is fairly low. For example, we're only now dropping support for MongoDB 4.0.

Reply:
I see; I moved the compatibility changes to its own section to separate it from the breaking changes